packages/lesswrong/themes/siteThemes/lesswrongTheme.ts

```diff
@@ -67,7 +67,7 @@ export const lessWrongTheme: SiteThemeSpecification = {
         fontSize: '.85em'
       },
       body2: {
-        fontSize: 15.1
+        fontSize: 15.08
```

Comment:
Slightly pedantic but this is the exact conversion of 1.16rem, and in the linked PR there are other px size going up to 2 decimal places so I changed these for consistency